app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<table class="chessboard">
+  <% 8.times do |row| %>
+    <tr>
+      <% 8.times do |column| %>
+        <% piece = Piece.where(x_cord: column, y_cord: row, game_id: @game).first %>
```

Comment:
It seems to work either way, but they were done differently on purpose: the game show page is concerned with the position of *all* the pieces at any given point, but on the piece show page, the only real concern is the piece you are about to move, and whose ID is listed in the path.
@ilyakrasnov - any input/guidance here regarding consistency vs. readability in this case?